src/main.ml

```diff
@@ -84,6 +84,6 @@ let () =
   let info =
     let doc = "Convert a directory structure into a standalone OCaml module that can serve the file contents without requiring an external filesystem to be present." in
     let man = [ `S "BUGS"; `P "Email bug reports to <mirage-devel@lists.xenproject.org>."] in
-    Cmd.info "ocaml-crunch" ~version:"2.1.0" ~doc ~man
+    Cmd.info "ocaml-crunch" ~version:{|%%VERSION%%|} ~doc ~man
```

Comment:
Could you use `dune-build-info` instead?

Reply:
Done!